db/migrate/20160217154334_create_stories.rb

```diff
@@ -0,0 +1,12 @@
+class CreateStories < ActiveRecord::Migration
+  def change
+    create_table :stories do |t|
+      t.integer :board_id
+      t.string  :title
+      t.text  :description
```

Comment:
I think about it and I think that you should add validation on max length for description. e.g 500 characters